src/test/exploit_tests.cpp

```diff
@@ -836,13 +836,13 @@ BOOST_AUTO_TEST_CASE(thinblock_tests)
         std::unique_ptr<CNode> dummyNode6Mem(new CNode(INVALID_SOCKET, addr1, "", true));
         CNode &dummyNode6 = *dummyNode6Mem;
 
-        // The number of tx bytes in this block is 3784 bytes.  In order for the node to be disconnected
+        // The number of tx bytes in this block is 3784 bytes howev.  In order for the node to be disconnected
```

Comment:
typo?

Reply:
these tests will all revert back to the original once i  fix the byte counting for the shared pointers.